app/_src/gateway/support/index.md

```diff
@@ -45,6 +45,9 @@ Customers with platinum or higher subscriptions may request fixes outside of the
 Kong supports the following versions of {{site.ee_product_name}}: 
 
 {% navtabs %}
+  {% navtab 3.3 %}
+    {% include_cached gateway-support.html version="3.2" data=site.data.tables.support.gateway.versions.33 eol="May 2024" %}
```

Comment:
```suggestion
    {% include_cached gateway-support.html version="3.3" data=site.data.tables.support.gateway.versions.33 eol="May 2024" %}
```

![image](https://github.com/Kong/docs.konghq.com/assets/23319190/c8903c97-117b-44ef-80e8-c0cd87e77b57)

Reply:
says 3.2 under the 3.3 tab